test/fixtures/suite/global-helper/01_test_one.bats

```diff
@@ -0,0 +1,3 @@
+@test "test one" {
+  [ "$(tail -1 /tmp/ghtestfile)" == "GH Test: test one" ]
```

Comment:
For this and the other tests:

* Use `BATS_TEST_SUITE_TMPDIR/ghtestfile`
* Instead of a subshell, use:
  ```bash
  [ "$(< "$BATS_TEST_SUITE_TMPDIR/ghtestfile")" == "GH Test: test one" ]
  ```